Cabal-tests/tests/CheckTests.hs

```diff
@@ -68,7 +68,7 @@ checkTest fp = cabalGoldenTest fp correct $ do
             -- Note: parser warnings are reported by `cabal check`, but not by
             -- D.PD.Check functionality.
             unlines (map (showPWarning fp) ws) ++
-            unlines (map show (checkPackage gpd Nothing))
+            unlines (map show (checkPackage gpd))
```

Comment:
`checkPackage` checks the package, does not need an additional configured `PackageDescription` no more.